app/portainer/settings/authentication/components/SaveAuthSettingsButton.tsx

```diff
@@ -0,0 +1,39 @@
+import { LoadingButton } from '@/portainer/components/Button/LoadingButton';
+
+export interface Props {
+  onSaveSettings(): void;
+  saveButtonState: boolean;
+  saveButtonDisabled?: boolean;
+  limitedFeatureId?: string;
+  limitedFeatureClass?: string;
+  className?: string;
```

Comment:
I don't see those used, why add them?

Reply:
original component has these props, which are utilised in the oauth, ldap and ad sections with this save button, so including them here was future proofing for when they will be migrated. I can always remove them and they can easily be added back later if you think thats the way to go :)

Reply:
Removed them :)